try-it.md

```diff
@@ -480,36 +474,24 @@ First, you must scale down the deployment of the `baremetal-operator` running
 in the cluster.
 
 ```sh
-kubectl scale deployment capm3-baremetal-operator-controller-manager -n metal3 --replicas=0
+$ kubectl scale deployment baremetal-operator-controller-manager -n baremetal-operator-system --replicas=0
 ```
 
 To be able to run `baremetal-operator` locally, you need to install
 [operator-sdk](https://github.com/operator-framework). After that, you can run
 the `baremetal-operator` including any custom changes.
 
 ```sh
-cd ~/go/src/github.com/metal3-io/baremetal-operator
-make run
+$ cd ~/go/src/github.com/metal3-io/baremetal-operator
+$ make run
```

Comment:
No big deal, but when including the `$` prompt, it would be more correct to use `console` or `shell` instead of `sh`. I noticed that this guide is using `sh` all the time and I think it would be nice to change it to `console`. It makes the syntax highlighting correct.

See https://stackoverflow.com/a/49004070 .